scrabble_sinatra.rb

```diff
@@ -0,0 +1,50 @@
+require 'sinatra'
+require 'CSV'
+require_relative 'lib/scrabble.rb'
+
+class ScrabbleSinatra < Sinatra::Base
+  FILE = "./support/words.csv"
```

Comment:
It never looks like you read from this file. Is it necessary for your app to run?